docs/openrpc.json

```diff
@@ -936,8 +943,16 @@
       "result": {
         "name": "The current client version.",
         "schema": {
-          "type": "string",
-          "pattern": "relay/[0-9]\\.[0-9]\\.[0-9]"
+          "anyOf": [
+            {
+              "type": "string",
+              "pattern": "relay\\/[0-9]+\\.[0-9]+\\.[0-9]+-SNAPSHOT"
```

Comment:
Q: if you added one for snapshot do we need one for rc versions?

Reply:
decided to write a check for the beginning of the string and that way we'll have one pattern for all